src/test/java/hudson/plugins/git/browser/GithubWebTest.java

```diff
@@ -64,15 +63,33 @@ public void testGetDiffLinkPath() throws IOException, SAXException {
     }
 
     @Test
-    public void testGetFileLinkPath() throws IOException, SAXException {
+    public void testGetFileLinkPath() throws IOException, SAXException, URISyntaxException {
```

Comment:
I've been advised by others that when a test throws a new exception type it is probably a good idea to reduce the list of exceptions thrown by the test rather than increase it.  I'm fine with this change if you feel to leave it this way, or you can change it to

```
throws Exception
```

Same rule does not apply to production code, just to test code.